pkg/client/client_test.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/rest"
+)
+
+type userAgentRoundTripper struct {
+	expectedUserAgent string
+	t                 *testing.T
+}
```

Comment:
Would it make sense to group the type with its method at the bottom of the file? Just for readability.

Reply:
I don't have a strong opinion on this. Went ahead and moved it.